src/BuildServer/BuildServerRequests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Oryx.BuildServer
+{
+    public class BuildServerRequests
+    {
```

Comment:
Would be nice to have a documentation link to a JSON sample

Reply:
of course we should .. good point